stable/etcd-operator/templates/_helpers.tpl

```diff
@@ -39,4 +39,37 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- define "etcd-restore-operator.fullname" -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s-%s" .Release.Name $name .Values.restoreOperator.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create the name of the etcd-operator service account to use
+*/}}
+{{- define "etcdOperator.serviceAccountName" -}}
```

Comment:
Please align with the existing templates which use dashes instead of camel case.